### PR TITLE
Add troubleshooting doc for admin-client-auth secret not found

### PR DESCRIPTION
**Pull request by bklei**
Fix for https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-4875

### CODE REVIEW

**Review by mharding-hpe — CHANGES_REQUESTED** [review]
Same changes as https://github.com/Cray-HPE/docs-csm/pull/1954

File: troubleshooting/known_issues/admin_client_auth_not_found.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+   ```text
+   kubectl get po -n services | grep keycloak-wait-for-postgres
+   keycloak-wait-for-postgres-1-pv85m                                0/2     Completed   0          15d
+   ```
+
+   1. If the `keycloak-wait-for-postgres-*` pod is not in a `Completed` state, resubmit the job:
+
+   ```bash
+   kubectl get job -n services keycloak-wait-for-postgres-1 -o json | jq 'del(.spec.selector)' | jq 'del(.spec.template.metadata.labels)' | kubectl replace --force -f -
```
Comment (kimjensen-hpe) [inline troubleshooting/known_issues/admin_client_auth_not_found.md:30]: I've seen systems where a newer job was created and the current one is is not keycloak-wait-for-postgres-1, might what to make this more generic with:
kubectl get job -n services -l app.kubernetes.io/instance=cray-keycloak-users-localize -o json | jq 'del(.spec.selector)' | jq 'del(.spec.template.metadata.labels)' | kubectl replace --force -f -

**Review by kimjensen-hpe — COMMENTED** [review]
One suggestion

File: troubleshooting/known_issues/admin_client_auth_not_found.md (hunk 1 above)
Comment (bklei) [reply troubleshooting/known_issues/admin_client_auth_not_found.md:33]: i like it -- will fix

Comment (bklei) [reply troubleshooting/known_issues/admin_client_auth_not_found.md]: fixed

**Review by kimjensen-hpe — APPROVED** [review]

**Review by phalseth-hpe — APPROVED** [review]

**Comment by mharding-hpe** [comment]
> @mharding-hpe added the prompts and your other changes in the final merged commit..

Those are the main branch style prompts, not 1.2 branch style commits.

For main branch, we put the prompt outside of the fenced code block.

For 1.2 branch, we still do it the old way, inside the code blocks.

**Comment by mharding-hpe** [comment]
https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-4882
https://github.com/Cray-HPE/docs-csm/pull/1956
https://github.com/Cray-HPE/docs-csm/pull/1957

**Pull request merged**